sciter.go

```diff
@@ -134,17 +134,17 @@ var (
 //  \param[in] data \b LPBYTE, pointer to data buffer.
 //  \param[in] requestId \b LPVOID, SCN_LOAD_DATA requestId.
 //  \return \b BOOL, TRUE if Sciter accepts the data or \c FALSE if error occured
-func (s *Sciter) DataReadyAsync(uri string, data []byte, requestId unsafe.Pointer) bool {
+func (s *Sciter) DataReadyAsync(uri string, data []byte, requestId C.HREQUEST) bool {
 	// args
 	var pData C.LPCBYTE
 	if len(data) > 0 {
 		pData = (C.LPCBYTE)(unsafe.Pointer(&data[0]))
 	}
 	curi := StringToWcharPtr(uri)
 	cdataLength := C.UINT(len(data))
-	crequestId := C.LPVOID(requestId)
+
 	// cgo call
-	ret := C.SciterDataReadyAsync(s.hwnd, curi, pData, cdataLength, crequestId)
+	ret := C.SciterDataReadyAsync(s.hwnd, curi, pData, cdataLength, requestId)

```

Comment:
This would cause Go to throw the following compile error:
`cannot use _cgo4 (type _Ctype_HREQUEST) as type _Ctype_LPVOID in argument to _Cfunc_SciterDataReadyAsync`

So perhaps you still need that `C.LPVOID` conversion?